project/AppDependencies.scala

```diff
@@ -1,21 +1,23 @@
+import play.core.PlayVersion
 import play.core.PlayVersion.current
 import sbt._
 
 object AppDependencies {
 
   private val catsVersion = "2.1.1"
 
-  val compile = Seq(
-    "uk.gov.hmrc"         %% "bootstrap-backend-play-27"       % "5.12.0",
-    "org.reactivemongo"   %% "play2-reactivemongo"             % "0.20.13-play27",
+  val compile: Seq[ModuleID] = Seq(
+    "uk.gov.hmrc"         %% "bootstrap-backend-play-28"       % "5.14.0",
+    "org.reactivemongo"   %% "play2-reactivemongo"             % "0.20.13-play28",
+    "org.reactivemongo"   %% "reactivemongo-play-json-compat"  % "0.20.13-play28",
```

Comment:
Did we need this dependency in the end? If it stays it could be worth extracting the shared version out into its own val